instrumentation/java11-http-client/java-http-client/README.md

```diff
@@ -0,0 +1,80 @@
+# Background on fork
+
+This module is a fork of the upstream [Java HTTP client instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/java-http-client/javaagent).
+This fork adds an HTTP Payload `AttributeExtractor` into the `InstrumenterBuilder`
+to facilitate collection of HTTP payloads of the request/response as an attribute on a span.
+It also adds a `ContextCustomizer` to add an instance of `ResponsePayloadBridge` into the
+OTeL context for passing HTTP response body content between instrumentation and the
+`AttributeExtractor`.
+
+This fork is necessary as the byte code advice in `HttpClientInstrumentation` directly accesses
+the static field on `JavaHttpClientSingletons` containing the instrumenter.
+There is no mechanism to extend the `InstrumenterBuilder` to replace aspects of what it provides,
+which is why this approach has been taken.
+
+Every update to OpenTelemetry JAVA SDK base version will require:
+- A re-copy of instrumentation changes from the [javaagent module](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/java-http-client/javaagent)
```

Comment:
looking at:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/commits/main/instrumentation/java-http-client/javaagent

it looks like this library is updated less than once a month.
do you think we should facilitate a process to get a notification for every change?

Reply:
It's something we can think about.

The upstream does a release every month, which may or may not include updates to the various instrumentations we're extending. We only need to update the instrumentations when we update the OTeL version. For now we can have a manual cadence of doing the updates